src/cloud-api-adaptor/entrypoint.sh

```diff
@@ -94,11 +94,11 @@ azure() {
 gcp() {
     test_vars GCP_CREDENTIALS GCP_PROJECT_ID GCP_ZONE PODVM_IMAGE_NAME
 
-    [[ "${PODVM_IMAGE_NAME}" ]] && optionals+="-gcp-image-name ${PODVM_IMAGE_NAME} "
+    [[ "${PODVM_IMAGE_NAME}" ]] && optionals+="-imageid ${PODVM_IMAGE_NAME} "
     [[ "${GCP_PROJECT_ID}" ]] && optionals+="-gcp-project-id ${GCP_PROJECT_ID} "
     [[ "${GCP_ZONE}" ]] && optionals+="-gcp-zone ${GCP_ZONE} "                         # if not set retrieved from IMDS
-    [[ "${GCP_MACHINE_TYPE}" ]] && optionals+="-gcp-machine-type ${GCP_MACHINE_TYPE} " # default e2-medium
-    [[ "${GCP_NETWORK}" ]] && optionals+="-gcp-network ${GCP_NETWORK} "                # defaults to 'default'
+    [[ "${GCP_MACHINE_TYPE}" ]] && optionals+="-instance-type ${GCP_MACHINE_TYPE} "    # default e2-medium
+    [[ "${GCP_NETWORK}" ]] && optionals+="-subnetid ${GCP_NETWORK} "                   # defaults to 'default'
```

Comment:
I see public doc mention subnets so this is better. What about the var? Should it be renamed to GCP_SUBNET_ID?

Reply:
I tried to avoid changing the env variable, because is public facing (Kustomization). But if we are ok with that, given we might not have that much users at this point, we can change it too.

Reply:
It's good to change now imho, since the GCP support is still new.

Reply:
Ok, I was about to change this, but I guess I will keep network here for now...

Currently, we are using the "network" logic, which let GCP choose the subnet. So here, is actually network, not subnet. And this was working fine with our tests. So I would be more comfortable to change to sub network on a different PR.